README.adoc

```diff
@@ -246,24 +260,6 @@ If the page you want to embed uses other URL arguments, you can specify any of t
 
 == Configure External Services
 
-=== Jaeger
```

Comment:
Answer is similar to @nrfox question about web_root config.

In general, we have been trying to centralize docs in the kiali.io website. I'm removing because these Jaeger configs are already documented in the website here: https://kiali.io/documentation/latest/distributed-tracing/

For the parts I kept (like Grafana), I couldn't find any mention on the website.

Additionally, since we have been trying to centralize documentation in the website, we tend to forget updating docs in this README. So, I wanted to de-duplicate docs as much as possible.

Reply:
Ok, perfect. Thanks!